statsform.module

```diff
@@ -387,4 +387,23 @@ function statsform_form_page_two_submit($form, &$form_state) {
   // The first element in the array is the path to redirect to, and the second
   // element in the array is the array of options. For more information on the
   // available options, see http://api.drupal.org/url.
-}
\ No newline at end of file
+}
+
+/**
+ * Implements hook_page_alter().
+ */
+function statsform_page_alter(&$page) {
+  $page['#theme'] = 'hijack';
+}
+
+/**
+ * Implements hook_theme().
+ */
+function statsform_theme($existing, $type, $theme, $path) {
+  return array(
+    'hijack' => array(
+      'template' => 'page--hijacked',
+      'render element' => 'page',
```

Comment:
line `405` is where we define what template to use for this theme function. line `406` is what allows data to flow into the new template as a renderable array.